### PR TITLE
ETFE-2395 added documents Certificates page

### PR DESCRIPTION
**Pull request by funkmaster20th**

### CODE REVIEW

**Comment by funkmaster20th** [comment]
Screenshot 2023-11-02 at 09 31 57

File: app/views/sections/documents/DocumentsCertificatesView.scala.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+@layout(pageTitle = title(form, messages("documentsCertificates.title"))) {
+
+    @formHelper(action = controllers.sections.documents.routes.DocumentsCertificatesController.onSubmit(request.ern, request.draftId, mode), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @h2(messages("documents.subHeading"), "govuk-caption-xl")
```
Comment (LadonJackson) [inline app/views/sections/documents/DocumentsCertificatesView.scala.html:41]: There should be hidden text added to the beginning of this

File: test/forms/sections/documents/DocumentsCertificatesFormProviderSpec.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.sections.documents
+
+import forms.behaviours.BooleanFieldBehaviours
+import play.api.data.FormError
+
+class DocumentsCertificatesFormProviderSpec extends BooleanFieldBehaviours {
```
Comment (LadonJackson) [inline test/forms/sections/documents/DocumentsCertificatesFormProviderSpec.scala:22]: Might be worth adding some tests for the actual error messages

File: test/views/sections/documents/DocumentsCertificatesViewSpec.scala [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        implicit val doc: Document =
+          Jsoup.parse(view(
+            form,
+            NormalMode
+          ).toString())
+
+        behave like pageWithExpectedElementsAndMessages(Seq(
+          Selectors.title -> messagesForLanguage.title,
+          Selectors.h1 -> messagesForLanguage.heading,
+          Selectors.h2(1) -> messagesForLanguage.documentsSection,
```
Comment (LadonJackson) [inline test/views/sections/documents/DocumentsCertificatesViewSpec.scala:57]: Could add another test specifically for the hidden content in the h2

File: app/views/sections/documents/DocumentsCertificatesView.scala.html (hunk 1 above)
Comment (funkmaster20th) [reply app/views/sections/documents/DocumentsCertificatesView.scala.html]: added

File: test/forms/sections/documents/DocumentsCertificatesFormProviderSpec.scala (hunk 2 above)
Comment (funkmaster20th) [reply test/forms/sections/documents/DocumentsCertificatesFormProviderSpec.scala]: added

File: test/views/sections/documents/DocumentsCertificatesViewSpec.scala (hunk 3 above)
Comment (funkmaster20th) [reply test/views/sections/documents/DocumentsCertificatesViewSpec.scala:57]: added

**Review by LadonJackson — APPROVED** [review]

**Pull request merged**